infrastructure/grpc-api/src/main/proto/io/timemates/api/users/types/User.proto

```diff
@@ -12,23 +12,26 @@ message User {
    * String identifier of 128 length. Can be null
    * if there is no avatar.
    */
-  optional string avatarId = 2;
+  oneof avatar {
+    string avatarId = 2;
+    string gravatarId = 3;
+  }
```

Comment:
you're missing optional (I am not sure is there any way to check whether oneof can be checked for optionality, but tryna google it)